app/Console/Commands/MigrateCouch.php

```diff
@@ -0,0 +1,129 @@
+    public function handle()
+    {
+
+        // if (! $this->confirm('Are you sure you want to proceed? This will delete ALL couchdb data in your environment. YOU SHOULD MAKE A BACKUP BEFORE RUNNING THIS COMMAND.')) {
+        //     return;
+        // }
+
+        $clientUsers = CouchDbHelper::client('_users');
+        $adm = CouchDbHelper::admin();
+
+        $this->info('✓ Removing every couchdb database');
+        $dbs = $clientUsers->listDatabases();
+        foreach ($dbs as $dbName) {
+            $db = CouchDbHelper::client($dbName);
+            $db->deleteDatabase();
+        }
+        $this->info('✓ Creating _users database');
+        $clientUsers->createDatabase();
+
+        /**
+         * Users.
+         */
+        $users = DB::table('users')->get();
+        $this->info('✓ Migrating '.count($users).' users');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($users));
+        $progress->start();
+        foreach ($users as $user) {
+            CouchUser::createFromUser($user);
+            $progress->advance();
+        }
+        $progress->finish();
+        $this->info('');
+        $this->info('');
+
+        /**
+         * Accounts.
+         */
+        $accounts = DB::table('accounts')->get();
+        $this->info('✓ Migrating '.count($accounts).' accounts');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($accounts));
+        $progress->start();
+        foreach ($accounts as $account) {
+            // create its database
+            $client = CouchDbHelper::getAccountDatabase($account->id);
+            $client->createDatabase();
+            CouchDbHelper::admin(CouchDbHelper::getAccountDatabaseName($account->id))->addDatabaseMemberRole(CouchDbHelper::getAccountDatabaseRoleName($account->id));
+
+            //setting note design documents
+            $favNotesFn = "function(doc) { if (doc.type === 'note' && doc.is_favorited === true) { emit(doc.created_at, null); } }";
+            $notesByContact = "function(doc) { if (doc.type === 'note') { emit([doc.contact_id, doc.created_at], null); } }";
+            $design_doc = (object) [
+                '_id' => '_design/notes',
+                'language' => 'javascript',
+                'views' => [
+                    'favorites'=> ['map' => $favNotesFn],
+                    'byContact' => ['map' => $notesByContact],
+                ],
+            ];
+            $client->storeDoc($design_doc);
+            $progress->advance();
+        }
+        $progress->finish();
+        $this->info('');
+        $this->info('');
+
+        $notes = DB::table('notes')->get();
+        $this->info('✓ Migrating '.count($notes).' notes');
+        $output = new ConsoleOutput();
+        $progress = new ProgressBar($output, count($notes));
+        $progress->start();
+        foreach ($notes as $note) {
+            $client = CouchDbHelper::getAccountDatabase($note->account_id);
```

Comment:
Is `$client` the right name for this variable?

Reply:
See http://php-on-couch.readthedocs.io/en/latest/api/couchclient/database.html#getting-started

But yeah, I guess we could name it `accountDatabase`.